tests/test_introduction_to_classes.py

```diff
@@ -248,4 +253,319 @@ def test_check_angles_sum_neg(test_angle1, test_angle2, test_angle3):
         assert str(e) == "Sum of given angles not equal to 180"
 
 
+# nicolas13sochi part
+def test_Animal6_description():
+    test_name6 = "Joffrey"
+    test_age6 = 5
+    test_is_hungry = True
+    animal = Animal6(test_name6, test_age6, test_is_hungry)
+    assert animal.description() == "{} + " " + {}".format(test_name6, test_age6)
+
+
+@pytest.mark.parametrize("test_animal_name_negative", [
+    None,
+    True,
+    False,
+    0,
+    "Joffrey"
+])
+@pytest.mark.parametrize("test_animal_age_negative", [
+    None,
+    True,
+    False,
+    0,
+    -1,
+    "",
+    6
+])
+@pytest.mark.parametrize("test_animal_is_hungry", [
+    None,
+    "Yes",
+    "No",
+    0,
+    -1,
+    ""
+])
```

Comment:
It'd be more readable if your list doesn't contain data structures (lists, tuples, dicts) and you put all values in 1 line:
```python
@pytest.mark.parametrize("test_animal_name_negative", [ None, True, False, 0, "Joffrey" ])
@pytest.mark.parametrize("test_animal_age_negative", [ None, True, False, 0, -1, "", 6 ])
@pytest.mark.parametrize("test_animal_is_hungry", [ None, "Yes", "No", 0, -1, "" ])
```

Reply:
please do this for other newly written tests.

Reply:
Should I do it for the all same situations in code? (I mean Sanya's part and my one)

Reply:
Well just for the new tests, readability counts!